init.sh

```diff
@@ -46,40 +49,48 @@ if [[ $DISTRO = "Ubuntu" ]]; then
     sudo apt install -y autossh sshpass git
 fi
 
-if [[ $DISTRO = "RHEL7" ]]; then
+if [[ ! -f initialized.txt && $DISTRO = "RHEL7" ]]; then
     rpm -qa | grep epel-release-7 || sudo yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm -y
     sudo yum upgrade -y
     sudo yum install -y autossh sshpass git
 fi
 
-if [[ $DISTRO = "RHEL8" ]]; then
+if [[ ! -f initialized.txt && $DISTRO = "RHEL8" ]]; then
     rpm -qa | grep epel-release-8 || sudo yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm -y
     sudo yum upgrade -y
     sudo yum install -y autossh sshpass git
 fi
 
+if [[ ! -f initialized.txt && $DISTRO = "AL2" ]]; then
+    echo "Detected Amazon Linux 2"
+    rpm -qa | grep epel-release-7 || sudo yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm -y
+    sudo yum upgrade -y
+    sudo yum install -y autossh sshpass expect git
```

Comment:
Don't need expect. 

Also, if the customer machine already has any of these already installed, you will end up failing the script (since initialized is not present). Is that right? 

If yes, then you will need a different way to handle the failure and continue to make progress.  and initialized will not serve it's purpose.

Reply:
The first time the init.sh is executed all these blocks will be run given that the other expression is also true.  

If the server already has any of those packages already installed, then this action is a no-op. The no-op won't exit the script.

You are right, I will remove the expect package since epel gives us the other packages.

Reply:
@AbsolutOD I don't think we need this PR as I feel this PR: https://github.com/lupfoss/lupmgr/pull/26 is smaller and is sufficient. Anything that PR is missing? Let's discuss on our call today.